todo_app/app.py

```diff
@@ -1,4 +1,6 @@
-from flask import Flask
+from flask import Flask, render_template, request, redirect
+from flask.helpers import url_for
+from .data.session_items import *
```

Comment:
minor, but it's considered bad practice to import * as it can pollute the namespace very easily, better to explicitly import on the things we need.
`from .data.session_items import get_items, add_item` etc....